deployments/data-hub/release-data-hub--stg.yaml

```diff
@@ -224,6 +227,125 @@ spec:
       # @TODO: legacy env var to be removed in the future - see Hazal or Daniel
       - name: AIRFLOW_NOTIFICATION_EMAIL_CSV_LIST
         value: ""
+    kubernetesPodTemplate:
+      resources:
+        requests:
+          memory: 4.9Gi
```

Comment:
If this is to run a single task, e.g. CSV pipeline, then the memory requirements could be lower?

Or we start with some specific tasks that I would expect to use high memory (some data science tasks).
Most of those don't run more than once a day, except for the `Extract_Keywords_From_Corpus` pipeline.

Reply:
I will have to observe the autoscaler too, just to make sure we have head room for potentially at lot more worker pods.

Reply:
It's probably better to start with the "big" jobs (e.g. `Data_Science_PeerScout_Recommend_Reviewing_Editors`) and start from the high watermark as you suggested.